src/LdapScimBridge.hs

```diff
@@ -172,14 +173,27 @@ instance Aeson.FromJSON (PhantomParent Level) where
 instance Aeson.FromJSON BridgeConf
 
 data MappingError
-  = MissingAttr Text
-  | MissingMandatoryValue Text
-  | WrongNumberOfAttrValues Text String Int
-  | CouldNotParseEmail Text String
-  deriving stock (Eq, Show)
+  = MissingMandatoryValue Text Text
+  | WrongNumberOfAttrValues Text Text String Int
+  | CouldNotParseEmail Text Text Text String
+  deriving stock (Eq)
+
+instance Show MappingError where
+  show = renderMappingError
```

Comment:
i took the short cut because i would have to mess with more types (we're trying to show a `[(SearchEntry, [MappingError])]` or something like that.

i'll fix it anyway.